TestClients/NICE.Identity.TestClient.NETFramework/Controllers/AccountController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Web;
+using System.Web.Mvc;
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Cookies;
+
+namespace NICE.Identity.TestClient.NETFramework.Controllers
+{
+	public class AccountController : Controller
```

Comment:
There seems to be an Auth0 integration in this Client project - can we not use the SDK (I'm assuming this is for .NET Core compatibility reasons?)?